### ISSUE TITLE
Copy Site: "Copy Site" appears near WP logo for a short time, and then disappears

### ISSUE THREAD

**Issue by kozer** [Automattic/wp-calypso#73046]
### Quick summary

After the load/reload of the domain step in Copy Site flow, "Copy Site" appears near the WP logo for a short time, and then disappears.

https://github.com/Automattic/wp-calypso/pull/72873#issuecomment-1419091272

This is happening due to how `StepContainer` [works](https://github.com/Automattic/wp-calypso/blob/trunk/packages/onboarding/src/step-container/index.tsx#L180)

### Steps to reproduce

1. Visit `/sites`
2. Copy a site.
3. On the first step (domain selection), reload the page.
4. Notice the title: appear and then disappears.

### What you expected to happen

Not show the title at all.

### What actually happened

the title appears and then disappears.

### Impact

One

### Available workarounds?

Yes, easy to implement

**Comment by danielbachhuber** [comment]
> This is happening due to how `StepContainer` [works](https://github.com/Automattic/wp-calypso/blob/trunk/packages/onboarding/src/step-container/index.tsx#L180)

@kozer What specifically is happening?

**Comment by kozer** [comment]
> > This is happening due to how `StepContainer` [works](https://github.com/Automattic/wp-calypso/blob/trunk/packages/onboarding/src/step-container/index.tsx#L180)
> 
> @kozer What specifically is happening?

`StepContainer` uses `should-sticky-nav-buttons` class and  `WordPressLogo` component when `shouldStickyNavButtons` is true (as we did in back-button pr). I think that this displays the component logo using a fixed position, on top of the original logo with the title, and for that reason, the title appears first and then disappears.

I created an issue and added the line where I think the problem lies.
